ingestion-beam/src/main/java/com/mozilla/telemetry/ContextualServicesReporter.java

```diff
@@ -70,31 +71,43 @@ public static PipelineResult run(ContextualServicesReporterOptions.Parsed option
         .failuresTo(errorCollections) //
         .apply(EmitCounters.of());
 
-    Set<String> aggregatedDocTypes = ImmutableSet.of("topsites-impression");
-    Set<String> perContextIdDocTypes = ImmutableSet.of("topsites-click");
+    Set<String> individualImpressions = ImmutableSet.of("topsites-impression");
+    Set<String> individualClicks = ImmutableSet.of("topsites-click");
+
     Set<String> unionedDocTypes = Stream
-        .concat(aggregatedDocTypes.stream(), perContextIdDocTypes.stream())
+        .concat(individualImpressions.stream(), individualClicks.stream())
         .collect(Collectors.toSet());
 
-    // Aggregate impressions.
-    PCollection<SponsoredInteraction> aggregated = requests
-        .apply("FilterAggregatedDocTypes", Filter.by((interaction) -> aggregatedDocTypes //
+    // Perform windowed click counting per context_id, adding a click-status to the reporting URL
+    // if the count passes a threshold.
+    PCollection<SponsoredInteraction> clicksCountedByContextId = requests
+        .apply("FilterPerContextIdDocTypes", Filter.by((interaction) -> individualClicks //
```

Comment:
can we change the name of this step? to something clearer about clicks? that will help debug the job when viewing the individual stages in the job graph.